static/js/upload.js

```diff
@@ -0,0 +1,44 @@
+var file = document.getElementById("file");
+file.onchange = function() {
+  if(file.files.length > 0) {
+    document.getElementById('filename').innerHTML = file.files[0].name;
+  }
+};
+
+function dataURItoBlob(dataURI) {
```

Comment:
Do we still need this?

Reply:
Yes, I think so. This is still needed for the correct serialization of the image for ajax